Sources/StreamChatSwiftUI/ChatChannelList/MoreChannelActionsView.swift

```diff
@@ -91,6 +106,24 @@ public struct MoreChannelActionsView: View {
         .onTapGesture {
             onDismiss()
         }
+        .fullScreenCover(isPresented: $isPresented) {
+            if let fullScreenView = presentedView {
+                ZStack(alignment: .topTrailing) {
+                    fullScreenView
+                    
+                    Button {
+                        isPresented = false
+                        presentedView = nil
+                    } label: {
+                        Image(uiImage: images.closeFilled)
+                            .customizable()
+                            .frame(height: 30)
+                            .padding(15)
+                            .foregroundColor(.black)
```

Comment:
We shouldn't use .black as a color. This makes the icon not visible in dark mode. Use a color from our color pallete (e.g. text). And please check if everything is clearly visible in dark mode.